src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/DataSetServiceTests.cs

```diff
@@ -0,0 +1,205 @@
+using System;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Repository;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Fixtures;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Utils.ContentDbUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.Tests;
+
+public class DataSetServiceTests
+{
+    public class GetDataSetTests : DataSetServiceTests
+    {
+        private readonly DataFixture _fixture = new();
```

Comment:
Changing `_dataFixture` to be declared outside of `GetDataSetTests` will mean its accessible to any other methods which get tested in this service which are added later on.